great_ape_safe/ape_api/univ3.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from brownie import interface, Contract
+from helpers.addresses import registry
+
+
+class UniV3:
+    def __init__(self, safe):
+        self.safe = safe
+
+        # contracts
+        self.NonfungiblePositionManager = safe.contract(
+            registry.eth.uniswap.NonfungiblePositionManager
+        )
+        self.pool = interface.IUniswapV3Pool(
+            registry.eth.uniswap.v3pool_wbtc_badger, owner=self.safe.address
+        )
```

Comment:
i suggest to rename this to `self.v3pool_wbtc_badger`

Reply:
ah also `owner=self.safe.account`. sometimes this is needed in case some native (eth) calls are made to owner.

Reply:
good one, just in case that we included more pools later on